infra/main.bicep

```
@@ -30,13 +30,34 @@ param solutionUniqueText string = take(uniqueString(subscription().id, resourceG
   'westeurope'
   'uksouth'
 ])
-param location string = 'australiaeast'
+param location string
 
 // Restricting deployment to only supported Azure OpenAI regions validated with GPT-4o model
 @allowed(['australiaeast', 'eastus2', 'francecentral', 'japaneast', 'norwayeast', 'swedencentral', 'uksouth', 'westus'])
-@metadata({ azd: { type: 'location' } })
-@description('Optional. Location for all AI service resources. This should be one of the supported Azure AI Service locations.')
-param azureAiServiceLocation string = 'australiaeast'
+@metadata({
+  azd : {
+    type: 'location'
+    usageName : [
+      'OpenAI.GlobalStandard.gpt-4o, 150'
+    ]
+  }
+})
+@description('Required. Location for all AI service resources. This should be one of the supported Azure AI Service locations.')
+param aiDeploymentsLocation string
```

As per our standard [PARAM-01](https://dev.azure.com/CSACTOSOL/CSA%20CTO%20Engineering/_wiki/wikis/CSA-CTO-Engineering.wiki/131/Bicep-standards?anchor=param-01), we decided to name this param as `azureAiServiceLocation`

```suggestion
param azureAiServiceLocation string
```

---

Updated it to azureAiServiceLocation